api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -808,6 +809,7 @@ type DNSSpec struct {
 	// It will be used to confgure ingress in the hosted cluster through the subdomain baseDomainPrefix.baseDomain.
```

Comment:
```suggestion
	// It will be used to configure ingress in the hosted cluster through the subdomain baseDomainPrefix.baseDomain.
```
unrelated type to this PR, but I think it's worth fixing here.

Reply:
happy to tag it anyway.